R/tm_g_bivariate.R

```diff
@@ -104,6 +109,10 @@
 #' if (interactive()) {
 #'   shinyApp(app$ui, app$server)
 #' }
+#'
+#'
+#' @export
+#'
```

Comment:
These empty spaces will be actually rendered out. Suggest to remove and keep the example code minimal.

Reply:
I mean the ones below `@export`. If you're referring to double empty lines above `@export`, I agree.

Reply:
yep, the ones above - I misclicked and highlighted more that I should :)